app.js

```diff
@@ -0,0 +1,160 @@
+let myLibrary = [];
+
+function setMyLibrary() {
+  if (localStorage.getItem("myLibrary") !== null) {
+    myLibrary = JSON.parse(localStorage.getItem("myLibrary"));
+    loadBooks();
+  } else {
+    myLibrary = [];
+  }
+}
+
+const booksContainer = document.querySelector(".books-container");
+const bookTitleInput = document.querySelector("#book-title");
+const bookDescriptionInput = document.querySelector("#book-description");
+const pagesInput = document.querySelector("#pages-number");
+const readStatusInput = document.querySelector("#read-status");
+const buttonInput = document.querySelector(".btn");
+const btnAddNewBook = document.querySelector(".btn-add-new-book");
+const newBookForm = document.querySelector(".new-book");
+
+buttonInput.addEventListener("click", createNewBook);
+
+function createNewBook() {
+  let title = bookTitleInput.value;
+  let desc = bookDescriptionInput.value;
+  let pages = pagesInput.value;
+  let status = readStatusInput.checked;
+
+  let book = {title, desc, pages, status};
+  // addBookToLibrary(book);
```

Comment:
- [ ] [REQUIRED] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)). The same applies to line 160.